src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -66,17 +72,19 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             originalCase.bulkScanCaseReference
         );
 
-        final List<String> warnings = updatedCaseValidator.getWarnings(newCase);
-
-        return new SuccessfulUpdateResponse(
-            new CaseUpdateDetails(
-                // This is just a sample implementation.
-                // You can use different event IDs based on the changes made to a case...
-                EVENT_ID,
-                newCase
-            ),
-            warnings
-        );
+        if (caseUpdateRequest.isAutomatedProcess && !warnings.isEmpty()) {
+            throw new InvalidCaseUpdateDetailsException(warnings);
```

Comment:
this could happen right after validation, no point in running the updates if this will happen next